.github/workflows/deploy.yml

```diff
@@ -0,0 +1,84 @@
+name: Deploy Backend
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  REGISTRY: ghcr.io
+  CONTAINER_NAME: c_backend
+
+jobs:
+  build-push:
+    name: Build and Push Docker Image
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Set Image Name to Lowercase
```

Comment:
Why do you need to set image name to lowercase?

Reply:
When building and pushing the docker image I run this command ` docker push REGISTRY/USERNAME/REPOSITORY:VERSION` and everything in this line has to be lowercase or it complains since I am pushing to GitHub Registry.

If I was using Docker Registry, I wouldn't need to make my username lowercase because, by default on push it makes it lowercase for you. 

I think this happened because GitHub registry is pretty new